### PR TITLE
[nri-bundle] readme missing integrations key in example

### PR DESCRIPTION
**Pull request by paologallinaharbur**
#### Is this a new chart
no
#### What this PR does / why we need it:
fix readme

#### Checklist
[Place an '[x]' (no spaces) in all applicable fields. Please remove unrelated fields.]
- [x] Chart Version bumped
- [x] Variables are documented in the README.md
- [x] Title of the PR starts with chart name (e.g. `[mychartname]`)

### CODE REVIEW

**Review by kang-makes — COMMENTED** [review]
It is missing also in [the README template](https://github.com/newrelic/helm-charts/blob/fix/bundleReadme/charts/nri-bundle/README.md.gotmpl#L80-L81)

File: charts/nri-bundle/README.md [hunk 1]
```diff
@@ -24,7 +24,7 @@ component separated, here is a list of components that this chart installs and w
 | [newrelic-k8s-metrics-adapter](https://github.com/newrelic/newrelic-k8s-metrics-adapter/tree/main/charts/newrelic-k8s-metrics-adapter) |  | (Beta) Provides a source of data for Horizontal Pod Autoscalers (HPA) based on a NRQL query from New Relic. |
 | [newrelic-logging](https://github.com/newrelic/helm-charts/tree/master/charts/newrelic-logging) |  | Sends logs for Kubernetes components and workloads running on the cluster to New Relic. |
 | [nri-prometheus](https://github.com/newrelic/nri-prometheus/tree/main/charts/nri-prometheus) |  | Sends metrics from applications exposing Prometheus metrics to New Relic. |
-| [newrelic-pixie](https://github.com/newrelic/helm-charts/tree/master/charts/newrelic-pixie) |  | Connects to the Pixie API and enables the New Relic plugin in Pixie. The plugin then sends data generated by a set of PxL scripts to New Relic. |
+| [newrelic-pixie](https://github.com/newrelic/helm-charts/tree/master/charts/newrelic-pixie) |  | Connects to the Pixie API and enables the New Relic plugin in Pixie. The plugin allows you to export data from Pixie to New Relic for long-term data retention. |
```
Comment (kang-makes) [inline charts/nri-bundle/README.md:27]: So I modified the template but not the README: https://github.com/newrelic/helm-charts/pull/846/commits/664266f8b8372620475b4d84a853e68fe5f9ca13

My bad, sorry.

**Review by kang-makes — APPROVED** [review]
LGTM

File: charts/nri-bundle/README.md (hunk 1 above)
Comment (paologallinaharbur) [reply charts/nri-bundle/README.md:27]: no worries, better that that modifying the readme and having it overwritten

**Pull request merged**